renku/core/models/provenance/agents.py

```diff
@@ -31,28 +31,39 @@
     context={
         'schema': 'http://schema.org/',
         'prov': 'http://www.w3.org/ns/prov#',
+        'rdfs': 'http://www.w3.org/2000/01/rdf-schema#'
     },
     frozen=True,
     slots=True,
 )
 class Person:
     """Represent a person."""
 
-    name = jsonld.ib(context='rdfs:label')
-    email = jsonld.ib(context='schema:email')
+    name = jsonld.ib(context='schema:name')
```

Comment:
Should we also add `validator` and `kw_only` here, like it was in `Creator`?
```
...
kw_only=True,
validator=instance_of(str),
...
```

Reply:
good point, thanks!